gulpfile.js

```diff
@@ -0,0 +1,44 @@
+var gulp                = require('gulp'),
+    livereload          = require('gulp-livereload'),
+    nodemon             = require('gulp-nodemon'),
+    shell               = require('gulp-shell'),
+    chalk               = require('chalk'),
+    paths,
+    nodemonServerInit;
+
+paths = {
+    server: 'core/server/',
+    client: 'core/client/'
+};
+
+nodemonServerInit = function () {
+    livereload.listen();
+
+    return nodemon({
+        script: 'index.js',
+        ext: 'js,json',
+        watch: [
+            'core/index.js',
+            paths.server + '**/*.js',
+            'core/built/assets/*.js'
+        ],
+        ignore: [
+            'core/client/*',
+            'core/server/test/*'
+        ]
+    }).on('restart', function () {
+        console.info(chalk.cyan('Restarting Ghost due to changes...'));
+        gulp.src('index.js')
+            .pipe(livereload());
+    });
+};
+
+// Starting Ember for development
+gulp.task('admin', shell.task(['cd ' + paths.client + ' && ember build --watch']));
+
+gulp.task('serve', function () {
```

Comment:
But that's exactly what I wanted to say... Sorry. I changed it to `server` already and I did this:

```
// Starting Ember server to run alone
gulp.task('admin', shell.task(['ember server --proxy http://127.0.0.1:3000'], {
    cwd: paths.client,
    env: {
        FORCE_COLOR: true // this is a fun little tidbit that will pass the colors from Ember to the main process
    }
}));

// Starting Ember build for development
gulp.task('_admin', shell.task(['ember build --watch'], {
    cwd: paths.client,
    env: {
        FORCE_COLOR: true // this is a fun little tidbit that will pass the colors from Ember to the main process
    }
}));
```